assets/charts/components/cert-manager/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: v0.16.1
+appVersion: v1.0.3
```

Comment:
Are there some steps for upgrading from 0.x to 1.x?

Reply:
Yes, I think there were some breaking change

Reply:
@knrt10 Don't think so.
https://cert-manager.io/docs/installation/upgrading/upgrading-0.16-1.0/#kubernetes-1-16-and-above

Reply:
cool.